index.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Dbu center</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+        integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay" crossorigin="anonymous" />
+<link rel="stylesheet" href="assets/css/index.css">
+</head>
+
+<body>
+    <header>
+        <nav class="secondary-header">
+            <ul class="secondary-nav">
+                <li class="top-nav-icon"><a href="#" ><i class="fab fa-facebook"></i> </a></li>
+                <li class="top-nav-icon"><a href="#"> <i class="fab fa-twitter"></i></a></li>
+                <li class="top-nav-icon"><a href="#"> <i class="fab fa-youtube"></i></a></li>
+                <li class="top-nav-icon"><a href="#">Login</a></li>
+                <li class="top-nav-icon"><a href="#"> English</a></li>
+            </ul>
+        </nav>
+        <div class="main-header" id="main-header">
+            <a href="index.html" class="logo"> 
+           <img src="./assets/images/logo.jpg" alt="Logo ">
+                <p class="company-name">Upper Tech</p>
+           
+            </a> 
+            <nav>
+                <ul class="main-nav hide-menu" id="main-nav" >
+                    <li class ="nav-item"><a href="about.html" class="nav-link">About </a></li>
+                    <li class="nav-item"><a href="#" class="nav-link">Services</a></li>
+                    <li class="nav-item"><a href="#" class="nav-link">achivements</a></li>
+                    <li class="nav-item"><a href="#" class="nav-link">Clients</a></li>
+                    <li class="nav-item"><a href="#" class="nav-link">Contact</a></li>
+                    <li class="nav-item"><a href="#" class="meetup">Meet up</a></li>
+
+                </ul>
+            </nav>
+           <div id ="hamburger" class="menu" >
```

Comment:
- [ ] Great Job on implementing the change. Please make sure that it is possible to navigate between pages, there is a link from `About` to `Home` and the opposite too
-
![n1](https://user-images.githubusercontent.com/63972670/169486003-6d572723-7cfd-4bd1-9381-ed3d7c5c7323.PNG)